src/Spinner.class.ts

```diff
@@ -78,7 +79,7 @@ export class Spinner extends EventEmitter {
       this.#color = (str: string) => str;
     }
     else {
-      this.#color = color in kleur ? kleur[color] : kleur.white;
+      this.#color = KColorKeys.includes(color) ? KColorKeys[color] : "white";
```

Comment:
It won't work?
https://github.com/TopCli/Spinner/blob/984bb7c47c9a95bb7ff90b5f518704c491cb76bb/src/Spinner.class.ts#L120

Here you are doing `"white"(str)`

Reply:
`this.#color` is a function, not the color value like `"white"`
before:  `kleur.white` => `kleur.white("boo some white text")`
after: `"white"` => `"white"("boo some white text")` (not good)